__tests__/restaurant.test.js

```diff
@@ -0,0 +1,175 @@
+
+const mockUser = {
+  firstName: 'Test',
+  lastName: 'User',
+  email: 'test@example.com',
+  password: '12345',
+};
+
+describe('Restaurant routes', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+  afterAll(() => {
+    pool.end();
+  });
+
+  it('/api/v1/restaurants should return a list of restaurants', async () => {
+    const res = await request(app).get('/api/v1/restaurants');
+    expect(res.status).toBe(200);
+    expect(res.body).toMatchInlineSnapshot(`
+      Array [
+        Object {
+          "cost": 1,
+          "cuisine": "American",
+          "id": "1",
+          "image": "https://media-cdn.tripadvisor.com/media/photo-o/05/dd/53/67/an-assortment-of-donuts.jpg",
+          "name": "Pip's Original",
+          "website": "http://www.PipsOriginal.com",
+        },
+        Object {
+          "cost": 3,
+          "cuisine": "Italian",
+          "id": "2",
+          "image": "https://media-cdn.tripadvisor.com/media/photo-m/1280/13/af/df/89/duck.jpg",
+          "name": "Mucca Osteria",
+          "website": "http://www.muccaosteria.com",
+        },
+        Object {
+          "cost": 2,
+          "cuisine": "Mediterranean",
+          "id": "3",
+          "image": "https://media-cdn.tripadvisor.com/media/photo-m/1280/1c/f2/e5/0c/dinner.jpg",
+          "name": "Mediterranean Exploration Company",
+          "website": "http://www.mediterraneanexplorationcompany.com/",
+        },
+        Object {
+          "cost": 2,
+          "cuisine": "American",
+          "id": "4",
+          "image": "https://media-cdn.tripadvisor.com/media/photo-o/0d/d6/a1/06/chocolate-gooey-brownie.jpg",
+          "name": "Salt & Straw",
+          "website": "https://saltandstraw.com/pages/nw-23",
+        },
+      ]
+    `);
+  });
+
+  it('GET api/v1/restaurants/:restId returns restaurant detail with nested reviews', async () => {
+    const res = await request(app).get('/api/v1/restaurants/1');
+    expect(res.status).toBe(200);
+    expect(res.body).toMatchInlineSnapshot(`
+      Object {
+        "cost": 1,
+        "cuisine": "American",
+        "id": "1",
+        "image": "https://media-cdn.tripadvisor.com/media/photo-o/05/dd/53/67/an-assortment-of-donuts.jpg",
+        "name": "Pip's Original",
+        "reviews": Array [
+          Object {
+            "detail": "Best restaurant ever!",
+            "id": "1",
+            "restaurant_id": "1",
+            "stars": 5,
+            "user_id": "1",
+          },
+          Object {
+            "detail": "Terrible service :(",
+            "id": "2",
+            "restaurant_id": "1",
+            "stars": 1,
+            "user_id": "2",
+          },
+          Object {
+            "detail": "It was fine.",
+            "id": "3",
+            "restaurant_id": "1",
+            "stars": 4,
+            "user_id": "3",
+          },
+        ],
+        "website": "http://www.PipsOriginal.com",
+      }
+    `);
+  });
+
+  const registerAndLogin = async () => {
```

Comment:
it'd be good to declare this function outside of the ```describe``` block up near ```mockUser```. not for functionality just stylistically